core/contextmenu_items.ts

```diff
@@ -279,7 +278,8 @@ export function registerDeleteAll() {
       }
       scope.workspace.cancelCurrentGesture();
       const deletableBlocks = getDeletableBlocks_(scope.workspace);
-      const eventGroup = idGenerator.genUid();
+      eventUtils.setGroup(true);
```

Comment:
If the user chooses cancel on the dialog, deleteNext_ is never run and grouping is never turned off. deleteNext_ was already calling setGroup() with the passed-in ID - why is this change necessary?

Reply:
This changes eliminates the dependency on idGenerator.  It also means that in Blockly there would be only one place which sets eventGroup -- so if we change the group format in the future, we won't need to change it in two places.